spyderlib/widgets/arraybuilder.py

```diff
@@ -256,7 +260,12 @@ def keyPressEvent(self, event):
             QDialog.keyPressEvent(self, event)
 
     def event(self, event):
-        if event.type() == QEvent.KeyPress and event.key() == Qt.Key_Tab:
+        """
+        Qt Override
+
+        Usefull when in line edit mode.
+        """
+        if event.type() == QEvent.KeyPress and event.key() == Qt.Key_Tab:  # pragma: nocover

```

Comment:
The pragma comment skips the coverage, right ? I don't think it should be there, because we could check the return value when given a specific event.